### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by J-avery32**

### CODE REVIEW

File: docs/Edit_Page.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Edit page
+## Link:
```
Comment (dylanmcreynolds) [inline docs/Edit_Page.md:2]: this subheading...should be removed? or is it half a thought ready to be complete?

File: docs/Edit_Page.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+In the upper right corner you can see a card that will display the date created and the creator. If the document has been edited then it will also display the date it was last edited at and the name of the person who edited it.
+
+## View past versions
+This button will take you to the ["Versioned view"](#) of this page.
+
+## The Document
+There will be a box that displays the actual content of the document. To edit the document, double click anywhere inside the box, or click on the pen icon in the upper left corner of the document. Hovering over any parenthetical references in the document should display a small pop up next to the cursor that will contain the APA citation of the corresponding reference. Clicking on any in-text citations will bring you down to 
+    
+ ## Editing the document
+ * ### Editor toolbar:
```
Comment (dylanmcreynolds) [inline docs/Edit_Page.md:21]: Can we remove all the colons at the end of headings?

File: docs/Compounds.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Compounds page
```
Comment (dylanmcreynolds) [inline docs/Compounds.md:1]: This page is formatted differently from Edit_Page in its use of sub-headings. I like Edit_page better. Please update this one to emulate that.

File: docs/Compounds.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+Link: mwet.lbl.gov/compounds
```
Comment (dylanmcreynolds) [inline docs/Compounds.md:2]: what is this?

File: docs/Edit_Page.md (hunk 1 above)
Comment (dylanmcreynolds) [reply docs/Edit_Page.md:2]: To a user, there is no thing called "Edit page". I think maybe you want to move this content into Compounds.md, and have two main sections..."View Mode" and "Edit Mode"

File: docs/Edit_Page.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+## Purpose
+This page displays a specific document and its corresponding references for the purpose of viewing and editing. 
+
+## Title
+The title of the document will be at the top of the page. You can edit the title by double clicking on it, or clicking the pencil icon. 
+
+## Metadata
+In the upper right corner you can see a card that will display the date created and the creator. If the document has been edited then it will also display the date it was last edited at and the name of the person who edited it.
+
+## View past versions
```
Comment (dylanmcreynolds) [inline docs/Edit_Page.md:14]: capitalization here is inconsistent... 

Sections Headers Should Be Capitalised Like This

File: docs/Edit_Page.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+This button will take you to the ["Versioned view"](#) of this page.
+
+## The Document
+There will be a box that displays the actual content of the document. To edit the document, double click anywhere inside the box, or click on the pen icon in the upper left corner of the document. Hovering over any parenthetical references in the document should display a small pop up next to the cursor that will contain the APA citation of the corresponding reference. Clicking on any in-text citations will bring you down to
```
Comment (dylanmcreynolds) [inline docs/Edit_Page.md:18]: "This box displays"

File: docs/Edit_Page.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    
+ ## Editing the document
+ * ### Editor toolbar:
+    The use of each button will be displayed by a tooltip if the the user hovers their mouse over the icon.
+ * ### Insert Reference:
+    This button will open up the "Insert Reference" pop up. Which is explained later in this document.
+* ### Save/Cancel:
+    The "Save" button will be disabled unless some change has been made to the document. The "Cancel" button will discard all changes made to the document since the last edit. 
+
+## Inserting in-text citations
```
Comment (dylanmcreynolds) [inline docs/Edit_Page.md:28]: This section should be merged with the similar section in Tutorials

File: docs/Edit_Page.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+This dialog box is used for inserting parenthetical citations into the text. It will insert these citations at where your cursor was located when you clicked the "Insert Reference" button. 
+
+You currently have two options. One, you can search the splash database itself for any references that may have been saved to it by other users. Secondly you can copy paste the DOI of the wanted reference into the "Create a Reference" field. This field will search a third party database for this specific reference. When you click "Create New," it will display the reference info with a "Create" button, that will save this reference to the splash database and insert it into the document. If this reference already exists in the splash database, you will be notified, it will display the citation and display an "Insert" button that will insert the citation into the document. Finally, if the DOI cannot be found in the third party database then the dialog box will inform you of that.
+
+## Saving with conflicts
```
Comment (dylanmcreynolds) [inline docs/Edit_Page.md:33]: This should be a "tutorial"

**Review by dylanmcreynolds — CHANGES_REQUESTED** [review]
This is a great start. Once you have addressed these comments, please read through for typos, capitalization, punctuation.

File: docs/Edit_Page.md (hunk 1 above)
Comment (J-avery32) [reply docs/Edit_Page.md:2]: The Link subheading has the link format to this part of the site.

One issue with moving this into Compounds.md is that later we will probably have pages that are not just compounds and so this is meant to be a generalized reference to editing any type of page

File: docs/Compounds.md (hunk 4 above)
Comment (J-avery32) [reply docs/Compounds.md:2]: The link is meant to link to this section of the site

File: docs/Edit_Page.md (hunk 7 above)
Comment (J-avery32) [reply docs/Edit_Page.md:28]: Do you mean the Inserting in text citations section or the save/cancel section?

File: docs/Edit_Page.md (hunk 1 above)
Comment (dylanmcreynolds) [reply docs/Edit_Page.md]: OK, then let's restructure so there's a "pages" reference and a "compounds" reference. Compounds says that it's a specialization of Pages, and links to pages.

File: docs/Edit_Page.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+This page displays a specific document and its corresponding references for the purpose of viewing and editing. 
+
+## Title
+The title of the document will be at the top of the page. You can edit the title by double clicking on it, or clicking the pencil icon. 
+
+## Metadata
+In the upper right corner you can see a card that will display the date created and the creator. If the document has been edited then it will also display the date it was last edited at and the name of the person who edited it.
+
+## View Past Versions
+This button will take you to the ["Versioned view"]() of this page.
```
Comment (dylanmcreynolds) [inline docs/Edit_Page.md:15]: this link is empty...do you intend for it to a reference that you will be writing?

**Review by dylanmcreynolds — APPROVED** [review]

**Pull request merged**